tedpermission/src/main/java/com/gun0912/tedpermission/TedPermissionActivity.java

```diff
@@ -186,7 +187,7 @@ private void checkPermissions(boolean fromOnActivityResult) {
         }
     }
 
-    private void permissionResult(ArrayList<String> deniedPermissions) {
+    private void permissionResult(List<String> deniedPermissions) {
```

Comment:
Is there a reason to change this?
It is a private function and there is no connection point with the outside.

Reply:
It was just base on `program to interfaces, not implementations`.

Although this request changed the listener used by the` ArrayList` to `List`, it was consistent under the same principle.